packages/next-templates/src/app/meldingen/page.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+import { Article, Page, PageContent } from '@utrecht/component-library-react';
+import { TopTaskNavigation } from '../components/TopTask/TopTaskNavigation';
+import { TopTaskLink } from '../components/TopTask/TopTaskLink';
+import {
+  UtrechtIconAfvalScheiden,
+  UtrechtIconMeldingKlacht,
+  UtrechtIconNummerbord,
+  UtrechtIconPaspoort,
+  UtrechtIconVerhuizen,
+  UtrechtIconWerken,
+} from '@utrecht/web-component-library-react';
+import '@/app/styling/css/detail-page.css';
+import '@utrecht/component-library-css';
+import '@utrecht/design-tokens/dist/index.css';
+import { ExampleHeader } from '@/components/ExampleHeader/ExampleHeader';
```

Comment:
Welke import type wil je gebruiken? `@/components` of `../components`? Nu is het inconsistent.